doc/ref_model/chapters/chapter03.md

```diff
@@ -13,21 +12,21 @@
 
-- Supporting any kind of workload specific requirements (e.g. regarding network acceleration or API access) might result in having to establish different silo of NFVIs for each workload type.
+- Supporting any kind of workload specific requirements (e.g. regarding network acceleration or API access) might result in having to establish different silo of cloud infrastructures for each workload type.
 - Synchronising the release cycles of a large set of different technologies will sooner or later lead to situations in which required upgrades cannot be applied easily due to incompatibilities.
 
-The abstraction model presented in this chapter specifies a common set of virtual infrastructure resources which NFVI will need to provide to be able to host most of the typical VNF/CNF workloads required by the operator community.
+The abstraction model presented in this chapter specifies a common set of virtual infrastructure resources which a cloud infrastructure will need to provide to be able to host most of the typical VNF/CNF workloads required by the operator community.
 
 Although a couple of explicit and implicit abstraction models (e.g. in the context of ETSI NFV) are already available, they fall short when addressing the following design principles:
-- **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) an NFVI needs to provide for hosting Telco workloads
+- **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) a cloud infrastructure needs to provide for hosting Telco workloads
 - **Separation of Concern:** the model should support a clear distinction between the responsibilities related to maintaining the network function virtualisation infrastructure and the responsibilities related to managing the various VNF workloads
 - **Simplicity:** the amount of different types of resources (including their attributes and relationships amongst one another) should be kept to a minimum to reduce the configuration spectrum which needs to be considered
-- **Declarative**: the model should allow for the description of the intended state and configuration of the NFVI resources for automated life cycle management
-- **Explicit:** the model needs to be rich enough to allow for a direct mapping towards the APIs of NFVIs for the instantiation of virtual infrastructure elements without requiring any additional parameters
+- **Declarative**: the model should allow for the description of the intended state and configuration of the cloud infrastructure resources for automated life cycle management
+- **Explicit:** the model needs to be rich enough to allow for a direct mapping towards the APIs of the cloud infrastructure for the instantiation of virtual infrastructure elements without requiring any additional parameters
```

Comment:
Not sure what it means to state, "allow for a direct mapping towards the APIs" and what parameters exist that the statement, "without requiring any additional parameters", is needed?

Reply:
@pgoyal01, I have changed it to 
- **Explicit:** the model needs to be rich enough to allow for the instantiation and the on-going operation of the cloud infrastructure

I guess that was the intent of this principle if first place.  Any further thoughts?